tests/tst_client.py

```diff
@@ -20,9 +20,9 @@ def test_marketo_client(client):
     assert client.API_LIMIT is None
     assert client.requests_timeout is None
 
-    client = MarketoClient('123-FDY-456', 'randomclientid', 'supersecret', 20, requests_timeout=10)
+    client = MarketoClient('123-FDY-456', 'randomclientid', 'supersecret', 20, requests_timeout=1.0)
```

Comment:
Was this file rename intentional? 🤯

Reply:
Seems like it was probably a mistake, but it was changed upstream:
https://github.com/jepcastelein/marketo-rest-python/commit/fd049a26b809afe003a0cee14e12402593e24a87